clients/fides-js/src/lib/events.ts

```diff
@@ -54,18 +54,22 @@ export const dispatchFidesEvent = (
   extraDetails?: FidesEventExtraDetails
```

Comment:
Now that you are automatically adding the `consentMethod` to these details, you should go through to all the existing usages of `dispatchFidesEvent` and ensure they don't provide their own version.

It should just be `preferences.ts` that currently does this (that was the problem, really- you could only get this `consentMethod` meta from the events fired by that module)

Reply:
ah I missed that, thanks!